core/tests/test_utils.py

```diff
@@ -1476,6 +1476,66 @@ def save_new_exp_with_states_schema_v35(self, exp_id, user_id, states_dict):
         )
         exp_summary_model.put()
 
+    def save_new_exp_with_states_schema_v36(self, exp_id, user_id, states_dict):
```

Comment:
Docstring is wrong -- it says v35 rather than v36.

Also -- why do we have to create a new one of these each time we do a state migration? That wasn't the case before and it would be better if it was not a requirement to do so.

Reply:
Hm, I created a new one for the new ```ExplorationMigrationAuditJobTests```, as it needs to create an exploration in the previous version to attempt migration to the next. Though instead of a function in test_utils, I will put it inline in ```exp_jobs_one_off_test.py```, so we aren't unnecessarily repeating a lot of code here.